modules/core/keeper/msg_server.go

```diff
@@ -527,6 +538,11 @@ func (k Keeper) Timeout(goCtx context.Context, msg *channeltypes.MsgTimeout) (*c
 func (k Keeper) TimeoutOnClose(goCtx context.Context, msg *channeltypes.MsgTimeoutOnClose) (*channeltypes.MsgTimeoutOnCloseResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	relayer, err := sdk.AccAddressFromBech32(msg.Signer)
```

Comment:
@AdityaSripal this was already implemented.

`Timeout` and `TimeoutOnClose` messages both call the same module callback, so I had to update them to get the code to compile